sql/parser/walk.go

```diff
@@ -86,6 +86,11 @@ func WalkExpr(v Visitor, expr Expr) Expr {
 	case *QualifiedName:
 		// Terminal node: nothing to do.
 
+	case Row:
```

Comment:
I'll file an issue about the above. In the meantime, add a test case to `TestFillArgs` which uses an explicit row: `ROW($1, $2, $3)`.